server/game/cards/events/01/unbowedunbentunbroken.js

```diff
@@ -41,13 +38,14 @@ class UnbowedUnbentUnbroken extends DrawCard {
             effect: ability.effects.cannotInitiateChallengeType(challengeType)
         }));
 
-        this.game.addMessage('{0} uses {1} to make {2} unable to initiate {3} challenges until the end of the phase', player, this, otherPlayer, challengeType);
+        this.game.addMessage('{0} plays {1} to make {2} unable to initiate {3} challenges until the end of the phase', player, this, otherPlayer, challengeType);
 
         return true;
     }
 
     cancel(player) {
-        this.game.addMessage('{0} cancel the resolution of {1}', player, this);
+        //I really want to get rid of stuff like this
```

Comment:
Things should always be cancellable and if they are then a message should be logged so people know if people are abusing it to cheat.  Also the code isn't the place to make these comments :)